### ISSUE TITLE
Still Adding Extra Intent to Switch Cases Inside #if/#endif

### ISSUE THREAD

**Issue by PompeiaPaetenari** [nicklockwood/SwiftFormat#876]
I updated to 0.47.12 in hopes I could remove the `// swiftformat:disable --indent tab` and its pair to enable after each, but the indentation still occurs:

Before:
```swift
    case .showPrompt:
      self = .showPrompt
    case let .showWebView(url):
      self = .showWebView(SafariConfig(url: url))

      // swiftformat:disable --indent tab
    #if DEBUG
    case .openEnvironment:
      self = .openEnvironment
    #endif
      // swiftformat:enable --indent tab
    }
```

After:
```swift
    case .showPrompt:
      self = .showPrompt
    case let .showWebView(url):
      self = .showWebView(SafariConfig(url: url))

    #if DEBUG
      case .openEnvironment:
        self = .openEnvironment
    #endif
    }
```

**Comment by nicklockwood** [comment]
@PompeiaPaetenari yes, sorry, this was only fixed for the case where you are using `--ifdef no-indent`. Perhaps that setting would work for you?

I guess it probably shouldn't indent here when set to `indent` either, but I'm not 100% sure what the correct behavior is tbh.

**Comment by PompeiaPaetenari** [comment]
When I applied that (`--ifdef no-indent`) it changes 54 files, so I am guessing that is not what we want to do.  I will ask the team, but the extra indentation in the switch being disabled does seem strange when we leave it in the rest of our code.

**Comment by PompeiaPaetenari** [comment]
So... folks want to keep the indentation for the rest of the code because it helps delineate the block.  However, for the code above, if I remove it and let it get indented, SwiftLint gets upset:
`Switch and Case Statement Alignment Violation: Case statements should vertically align with their enclosing switch statement. (switch_case_alignment)`

**Comment by nicklockwood** [comment]
OK, well I'll consider whether it makes sense to special-case this. It seems weird not to indent if that's what you've asked for, but OTOH I can't really imagine anyone would want to indent cases to different levels.
